docs/developer-docs/setup/install/index.mdx

```diff
@@ -105,16 +105,30 @@ generateResolvConf = false`}
 
 ## What gets installed
 
-The SDK installation script installs several components in default locations on your local computer. The following table describes the development environment components that the installation script installs:
+The IC SDK installation script installs several components in default locations on your local computer. The following table describes the development environment components that the installation script installs:
 
 | Component    | Description                                                                                        | Default location                              |
 |--------------|----------------------------------------------------------------------------------------------------|-----------------------------------------------|
-| dfx          | DFINITY execution command-line interface (CLI)                                                     | `/usr/local/bin/dfx`                          |
+| dfx          | Command-line interface (CLI)                                                     | `/usr/local/bin/dfx`                          |
+| cdk-rs       | Rust CDK                                                       | `/usr/local/bin/dfx`                          |
 | moc          | Motoko runtime compiler                                                                            | `~/.cache/dfinity/versions/<VERSION>/moc`     |
 | replica      | Internet Computer local network binary                                                             | `~/.cache/dfinity/versions/<VERSION>/replica` |
 | uninstall.sh | Script to remove the SDK and all of its components                                    | `~/.cache/dfinity/uninstall.sh`               |
 | versions     | Cache directory that contains a subdirectory for each version of the SDK you install. | `~/.cache/dfinity/versions`                   |
 
+## SDK vs CDK vs `dfx`
+
+There are a few components above worth expanding on:
+
+1. **IC SDK** - This is the SDK created by DFINITY for creating and managing canisters. This is just one SDK in ICP community and a common entry point for developers.
+
+2. **Canister Development Kit (CDK)** - A CDK is an adapter used by the IC SDK so a programming language has the features needed to create and manage canisters. 
+The IC SDK comes with a few CDKs already installed for you so you can use them in the language of yoru choice. That is why there is a [Rust CDK](https://github.com/dfinity/cdk-rs), [Python CDK](https://demergent-labs.github.io/kybra/), 
+[TypeScript CDK](https://demergent-labs.github.io/azle/), etc... Since CDKs are components used the SDK, some developer choose to use the CDK directly (without the IC SDK), 
```

Comment:
```suggestion
[TypeScript CDK](https://demergent-labs.github.io/azle/), etc... Since CDKs are components used by the SDK, some developers choose to use the CDK directly (without the IC SDK), 
```